COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Public Slack](#public-slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to provide references and information regarding
+the communication channels for the OpenSearch Project. All communication is
+subject to the [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md). Please see
+[CONTRIBUTING](CONTRIBUTING.md) if you're interested in contributing to the
+project.
+
+## Public Slack
+
+OpenSearch has a workspace on [Slack](https://opensearch.slack.com) to provide
+public communication channels for all people interested in OpenSearch.
+
+The following guidelines include a Getting Started for steps to register and
+setup the workspace along with tips for using the workspace. Please read through
+these guidelines carefully and thoroughly. All communications and behavior on
+the public slack workspace is subject to the 
+[OpenSearch Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Getting Started
+
+_Join Us_
+
+  * Register your account at https://opensearch.slack.com. You will be required
+    to enroll in two-factor authentication to ensure the best possible security
+    of your account.
+
+_Update Profile_
+
+  * Add your interests or keywords in the “What I do” section of your profile 
```

Comment:
This would come across better as "You MAY" Current reading feels like "You MUST" (RFC 2119 meaning implied.) 

There may also be folks who wish to be identified as individuals, or may be prohibited by their employer from identifying their affiliation.

Reply:
:+1: